api/cmd/query/main.go

```
@@ -57,13 +59,21 @@ func parseopts() opts {
 		"Signing key used for response authorization tokens",
 		"key",
 	)
+	port := getopt.IntLong(
```

Why parse as int with getopt, when you want the port as string anyways?

---

Original idea was that port is a number. So lets parse it as a number! 
But if you are wondering I probably better change it to a string. It will fail fast enough anyway.

---

Not a big deal, it's probably gonna panic with a descriptive message in both cases